tests/unit/test_manager.py

```diff
@@ -0,0 +1,69 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
perhaps test_lifecycle_manager.py?

Reply:
I think it's a better name, I also should rename `manager.py` to `lifecycle_manager.py`. This will be done in a separate PR.